### PR TITLE
Publish library symbols to dotnetbuilds

### PR DESCRIPTION
**Pull request by mmitche**
Not sure why this didn't switch over to dotnetbuilds a ways back.

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs [hunk 1]
```diff
@@ -131,8 +131,8 @@ public enum BuildQuality
         {
             (TargetFeedContentType.Package, FeedDotNet31Shipping, AssetSelection.ShippingOnly),
             (TargetFeedContentType.Package, FeedDotNet31Transport, AssetSelection.NonShippingOnly),
-            (InstallersAndSymbols, FeedForInstallers),
-            (TargetFeedContentType.Checksum, FeedForChecksums),
+            (InstallersAndSymbols, FeedStagingForInstallers),
+            (TargetFeedContentType.Checksum, FeedStagingForChecksums),
```
Comment (riarenas) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:135]: Should we also change LN 150 to use this one? and the internal feeds to use the internal equivalent? Looks like we didn't change any3.1 ones, but I can't find whether that was intentional.

Also should we change the .NET 5 ones, or are those dead dead? I never remember.

Comment (mmitche) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:133]: 3.1 and 5.0 never got the updates to pull from dotnetbuilds. I looked at the blazor lines and it turns out the variables aren't even referenced, so I just deleted them.

Comment (riarenas) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs:133]: This is changing the 3.1 feeds isn't it? Maybe this change is supposed to go here? https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs#L955

**Comment by mmitche** [comment]
Okay better now.

**Review by riarenas — APPROVED** [review]

**Pull request merged**